src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -450,6 +450,18 @@ or reader
   | `OTEL_EXPORTER_OTLP_TIMEOUT`  | `TimeoutMilliseconds`                 |
   | `OTEL_EXPORTER_OTLP_PROTOCOL` | `Protocol` (`grpc` or `http/protobuf`)|
 
+  The following environment variables can be used to configure mTLS
+  (mutual TLS) authentication (.NET 8.0+ only):
+
+  | Environment variable                             | `OtlpMtlsOptions` property    | Description                           |
+  | -------------------------------------------------| ----------------------------- | ------------------------------------- |
+  | `OTEL_EXPORTER_OTLP_CERTIFICATE`                 | `CaCertificatePath`           | Path to CA certificate file (PEM)     |
+  | `OTEL_EXPORTER_OTLP_CLIENT_CERTIFICATE`          | `ClientCertificatePath`       | Path to client certificate file (PEM) |
+  | `OTEL_EXPORTER_OTLP_CLIENT_KEY`                  | `ClientKeyPath`               | Path to client private key file (PEM) |
+  | `OTEL_EXPORTER_OTLP_CLIENT_KEY_PASSWORD`         | `ClientKeyPassword`           | Password for encrypted client private key file (PEM) |
+  | `OTEL_EXPORTER_OTLP_CERTIFICATE_REVOCATION_MODE` | N/A                           | Certificate revocation mode (`Online`, `Offline`, or `NoCheck`). Default: `Online` |
+  | `OTEL_EXPORTER_OTLP_CERTIFICATE_REVOCATION_FLAG` | N/A                           | Certificate revocation flag (`ExcludeRoot`, `EntireChain`, or `EndCertificateOnly`). Default: `ExcludeRoot` |
```

Comment:
We can only offer configuration options that are [defined in the specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md#configuration-options). These options are not currently defined. If they are something you want support for, then they need to be driven through the specification first.

Reply:
@alanwest I created the issue for now: https://github.com/open-telemetry/opentelemetry-specification/issues/4580 and will remove them for now.